rpcs3/Emu/RSX/Overlays/HomeMenu/overlay_home_menu_components.cpp

```diff
@@ -0,0 +1,65 @@
+#include "stdafx.h"
+#include "overlay_home_menu_components.h"
+
+namespace rsx
+{
+	namespace overlays
+	{
+		home_menu_entry::home_menu_entry(const std::string& text)
+		{
+			std::unique_ptr<overlay_element> text_stack = std::make_unique<vertical_layout>();
```

Comment:
Again, these can be objects.
```C++
// In header file
vertical_layout m_text_stack;
```

Reply:
add_element takes a unique_ptr and moves it